modules/platform/nuxeo-platform-comment/nuxeo-platform-comment-api/src/main/java/org/nuxeo/ecm/platform/comment/api/CommentImpl.java

```diff
@@ -270,6 +270,9 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return new ToStringBuilder(this).append("id", getId()).toString();
+        return new ToStringBuilder(this).append("id", getId())
+                                        .append("author", getAuthor())
+                                        .append("text", getText())
+                                        .toString();
```

Comment:
I am wondering if we should add the `author` and `text` to `toString` method in the case where we log a sensitive data I mean. what each person says :)

Reply:
Good interrogation, I don't have any clue on this. Maybe the @nuxeo/platform team?

Reply:
A,greed the id is enough imho.

Reply:
Agree with you @troger, that's I didn't have any clue...

In the end, I rollbacked.